sdk/test-utils/recorder/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2020-04-30
+
+- Since Mocha 7.0.0, Mocha behaves as follows: "When conditionally skipping in a it test, related afterEach hooks are now executed"
```

Comment:
```suggestion
- Since Mocha 7.0.0, Mocha behaves as follows: "When conditionally skipping in the `it` test, related afterEach hooks are now executed"
```